portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,26 @@ function randomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = Fact;
 }
+function getName(){
```

Comment:
[Style](https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace): Please add a blank line before and add a blank space before the curly brace.

```suggestion

function getName() {
```